doc/operator-user-guide.md

```diff
@@ -20,6 +20,7 @@
 * [3scale installation Backup and Restore using the operator (in *TechPreview*)](operator-backup-and-restore.md)
 * [Application Capabilities (in *TechPreview*)](operator-application-capabilities.md)
 * [APIManager CRD reference](apimanager-reference.md)
+    * [APIManager CR samples](cr_samples/apimanager/)
```

Comment:
We should include a link to the one from config/samples which is the simple one

Reply:
Added